srcpkgs/immortal/template

```diff
@@ -13,7 +13,8 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://github.com/immortal/immortal"
 distfiles="https://github.com/immortal/immortal/archive/${version}.tar.gz"
-checksum=e31d5afb9028fb5047b5a2cc5f96c844f6480d600643a12075550f497e65f5cb
+checksum=a343581dbe58fb0faa1c65b233a067820d8d5ecefc9726da5ad3ef979a2a0b08
+make_check=ci-skip # tests pass locally
```

Comment:
That's not a reason, ci-skip already tells us that the tests pass locally.

Reply:
Oh thats right... I've enabled them now. How would you describe this?

Reply:
or should i just `make_check=ci-skip` without a comment?

Reply:
Updated it now :)

Reply:
Tests also fail locally for me, so it shouldn't be ci-skip and it's not caused by being root.

Reply:
And now they pass after a clean.